autogen/README.md

```diff
@@ -95,70 +95,8 @@ Then perform the following commands on the root folder:
 - `terraform apply` to apply the infrastructure build
 - `terraform destroy` to destroy the built infrastructure
 
-
+<!-- Since the inputs and outputs will not be autogenerated in this directory, there is no reason to store them here. -->
```

Comment:
I think removing the inputs and outputs from this file is sufficient to make this point. This comment will be copied to the generated READMEs which is confusing when viewing the source. 🙃

Reply:
Done.